core/src/main/java/com/linecorp/armeria/client/logging/ContentPreviewingClient.java

```diff
@@ -54,6 +56,9 @@
  */
 public final class ContentPreviewingClient extends SimpleDecoratingHttpClient {
 
+    private static final AttributeKey<Boolean> CONTENT_PREVIEWING_SET =
+            AttributeKey.valueOf(ContentPreviewingClient.class, "CONTENT_PREVIEWING_SET");
```

Comment:
Shouldn't it be `SETTING_CONTENT_PREVIEW`? Technically we did not set any preview they setting true.

Reply:
Thanks, fixed. 😉